src/agent/block_store_services/block_store_fs.js

```diff
@@ -249,6 +253,88 @@ class BlockStoreFs extends BlockStoreBase {
         }
     }
 
+
+    /**
+     * @param {string[]} block_ids
+     * @param {string} storage_class
+     * @returns {Promise<{ moved_block_ids: string[] }>}
+     */
+    async _move_blocks_to_storage_class(block_ids, storage_class) {
+        if (storage_class === 'FS_ARCHIVE') {
+            return this._move_blocks_to_fs_archive(block_ids);
+        }
+        if (storage_class === 'STANDARD') {
+            // Nothing to do for now
+            return Promise.resolve({ moved_block_ids: [] });
+        }
```

Comment:
Is this api being called when moving the chunks back to first tier?
If so it's better if we actually trigger a recall here, because then our actual reads will have a chance to already find the data ready.

Reply:
Yes it is. I also considered doing the same but then decided to skip doing it in this PR and rather address it in another PR which:
1. Removes the responsibility of "archiving" from the write flow because TTL worker can take care of that.
2. Adds the recall here.

Basically a PR dedicated to Read/Write flow. WDYT?

Reply:
sure, that makes sense. lets keep that focus.